CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# Changelog
+
+The format of this document is inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and the project follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- This is a comment, you won't see it when GitHub renders the Markdown file.
+
+When releasing a new version:
+
+1. Remove any empty entry (those with `_None._`)
```

Comment:
I should have said `section`, not `entry`, in my suggestion from my comment before, sorry 😅 

```suggestion
1. Remove any empty section (those with `_None._`)
```